go-get/README.md

```diff
@@ -25,6 +25,28 @@ The script does not require Go in the host system.
 }
 ```
 
+Yaml example:
+```yaml
+app-id: writeas-cli
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.golang
+command: echo "Done"
+modules:
+  - name: writeas
+    buildsystem: simple
+    build-options:
+      append-path: /usr/lib/sdk/golang/bin
+      env:
+        GOBIN: /app/bin
+      build-args:
+        - --share=network
+    build-commands:
+      - "go env -w GOPATH=$PWD; go env -w GO111MODULE=off; go get github.com/writeas/writeas-cli/cmd/writeas"
```

Comment:
Valid point. I moved `GO111MODULE` to `build-options.env`, with the GOPATH it seemingly needs to be set at runtime, as if I move it out of `build-commands` I get the following error:

go: GOPATH entry is relative; must be absolute path: "$PWD".

Reply:
True: I am happily configuring the GOPATH as the value which is the working directory: As I am not the original author I was aiming to keep the second example aligned with the first one.

Seeing that this is a kind of official example, I am happily getting it as good as possible.

I see a benefit in using $PWD here as it means there are fewer code changes needed as to use this in practice: as it stands now I understand the only change needed is the `go get` URL.

If you know your way around go, do you know if we should use the `go install` rather than the `go get` here?
I understand that the language itself has changed with the recent versions.

Reply:
Hello @gasinvein, the usage of $PWD is now replaced with setting the env of the module to `/run/build/writeas` as you suggested.